data.json

```diff
@@ -95,6 +95,14 @@
     "image": "https://partners.npmjs.com/weekly/weekly30/socks-1100x.jpg",
     "tags": ["clothing"]
   },
+  {
+    "name": "OpenBazaar",
+    "difficulty": "medium",
+    "description": "Help improve our code and earn a limited edition shirt.",
+    "reference": "https://twitter.com/openbazaar/status/1053016648347717632",
```

Comment:
Would be better to give the reference to their blog post: https://openbazaar.org/blog/Hacktoberfest-with-OpenBazaar-Help-Make-Trade-Free-Get-Rewards/
:cake: :sparkles:

Reply:
Agreed. But I felt that the blog wasn't clear about whether developers are eligible for t-shirts. Their tweet cleared the ambiguity.
Open to modifying the PR if required.

Reply:
+1 for using the blog post url.